### DISCUSSION TITLE
Unable to download files when in reverse proxy [May 31]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5843] [Discord Threads]
> 🙋‍♂️ **rjsmiley**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### To resolve the issue regarding reverse proxy, set `NC_PUBLIC_URL` environment varibale with your URL.  
#  
> 🙋‍♂️ **rjsmiley**  
#### <@682854952657616902>  Thank so much! That seems to have resolved the issue. I appreciate your help.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1113216650578251796)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
